common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -92,12 +89,13 @@
                                 for="submit-google-search"
                                 type="submit">
                             <i class="right-arrow__icon"></i>
+                            <span class="u-h">Submit your search</span>
```

Comment:
What about changing text to "Search with Google". I remember @zeftilldeath and Gustavo talking about this last week